### PR TITLE
112 author doc

### LINKED ISSUE #112: Author Documentation
documentation of blocks, templates, fragments and metadata for the author

### PR DESCRIPTION
**Pull request by chhaack**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs (without age check):
- Before: https://main--sazerac-wheatleyvodka--hlxsites.hlx.page/?noAgeCheck=yes
- After: https://112-author-doc--sazerac-wheatleyvodka--hlxsites.hlx.page/?noAgeCheck=yes

Test URLs (with age check):
- Before: https://main--sazerac-wheatleyvodka--hlxsites.hlx.page/
- After: https://112-author-doc--sazerac-wheatleyvodka--hlxsites.hlx.page/

### CODE REVIEW

File: docs/blocks/ageverification.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Age Verification
+The age verification block is automatically integrated into every page delivered if the cookie sazAgeOK is missing.<br>
```
Comment (karlpauls) [inline docs/blocks/ageverification.md:2]: Can you try to explain it a bit more - just another sentence or two about how the cookie is working and what happens with accepting vs rejecting the question (redirect etc.).

File: docs/blocks/buy-template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Buy
+
+Definition of the output of the buy block. Authoring has to done via templates/buy.docx.
```
Comment (karlpauls) [inline docs/blocks/buy-template.md:3]: "has to done via" => "has to be done via"

Might be worthwhile to mention explicitly that these are files in Sharepoint (in the other cases as well).

File: docs/blocks/pocketguide.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+## Pocket Guide
+
+download block for the pocket guide pdf
```
Comment (karlpauls) [inline docs/blocks/pocketguide.md:3]: Maybe it shouldn't say "download" but "insert" or even just remove it and just say: "block for"

File: docs/blocks/columns.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+## Columns
```
Comment (karlpauls) [inline docs/blocks/columns.md:1]: Can you double check if we have different variants of the columns block we use? I think we have at least the case where it is making the image a background image and another one where we can reverse the way we break the columns.

**Review by karlpauls — CHANGES_REQUESTED** [review]
Looks good, I added some minor comments - could you have a look?

File: docs/sections-metadata.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Section Metadata
```
[the remark this answers is not in the document]
Comment (chhaack) [reply docs/sections-metadata.md:1]: only found column block using section metadata at the moment

File: docs/blocks/recipes.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+## Recipes
+
+Recipes with ingredients and directions including image. Every Recipe is defined as single document under
+/coctails/ and uses the /template/cocktail.docx template for generating the html (Sharepoint).
+
+### Content Structure
+
+title (h1)<br/>image<br/>text (h2)<br/>list<br/>text (h2)<br/>list<br/>
+
```
Comment (RemoLiechti) [inline docs/blocks/recipes.md:9]: created by missing, see some other cocktails

**Comment by RemoLiechti** [comment]
Issues found:

**For the cocktail detail page (receipe)**
I added a proposal for the missing creator information to the receipes.md. Use the flatiron flip cocktail for details: https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B6090EA25-73A7-48BF-824B-7599FC820F44%7D&file=flatiron-flip.docx&action=default&mobileredirect=true

1. add the creator information to the documentation
2. update the three screenshots with the flatiron flip instead
3. describe the templating mechanism, showing the content of the template, where the template is stored, how to change it. The template needs to be published individually to reflect any changes online.

**different ways of describing**
Various ways on how content shall be structured found. This may confuse the reader, i.e if he needs to put special characters. Stick to one consistent way of notation across all documentation.

- (img) vs image vs [image]
- list vs (ul)(li)entry(li)entry vs (List with links)
- (a) vs [links] vs (embed link) vs link
- text vs [text]
- (h1) vs title

**Review by karlpauls — APPROVED** [review]

**Pull request merged**